index.d.ts

```diff
@@ -612,6 +612,8 @@ declare namespace Eris {
     systemChannelFlags: 0 | 1 | 2;
     systemChannelID?: string;
     verificationLevel?: number;
+    discoverySplash: string;
+    features: string[];
```

Comment:
Both should be alphabetically positioned and both should also be optional
```suggestion
    discoverySplash?: string;
    features?: string[];
```